tests/test_wave_05.py

```diff
@@ -117,34 +110,26 @@ def test_delete_goal(client, one_goal):
     assert response.status_code == 200
     assert "details" in response_body
     assert response_body == {
-        "details": 'Goal 1 "Build a habit of going outside daily" successfully deleted'
+        "details": 'goal 1 "Build a habit of going outside daily" successfully deleted'
```

Comment:
I would expect you to update your Goal Delete route's behavior rather than changing the test...